storage-node/src/services/queryNode/schema.graphql

```diff
@@ -957,6 +962,9 @@ type DistributionBucketOperatorMetadata implements BaseGraphQLObject {
   nodeLocation: NodeLocationMetadata
   nodeLocationId: String
 
+  """Optional node operational status"""
+  nodeOperationalStatus: NodeOperationalStatus
```

Comment:
I think this can be hard to query because of QN nested filtering limitations. I think we need to design a schema in a way that I can easily fetch all the active operators. If I'm not mistaken with the current schema the query would have to be (paraphrasing, I don't remember the query syntax for checking specific union type):
```gql
query {
  distributionBucketOperators(
    where: { metadata: { nodeOperationalStatus: { status: "OKAY" } } }
  ) {
    id
  }
}
```
And that filter would be too deep for QN to execute. Same applies to storage buckets

Reply:
Also, would it be possible to make operational status required part of metadata? Because empty operational status currently would also mean "Okay" but it would be hard to query it I think

Reply:
I think you would not have that problem with storage squid's graphql server